dotCMS/src/main/java/com/dotcms/util/AnnotationUtils.java

```diff
@@ -243,4 +241,31 @@ public static boolean isMethodAnnotatedBy(final Method method, final Class<? ext
         return null;
     } // getMethodAnnotatedBy.
 
+    /**
+     * Get all methods annotated by annotationType, not looking for super class. Just the subclass
+     * @param object {@link Object}
+     * @param annotationType {@link Class}
+     * @return Set returns an immutable set with the methods annotated by annotationType
+     */
+    public static Set<Method> getMethodsAnnotatedBy (final Object object,
+                                                     final Class<? extends Annotation> annotationType) {
+
+        final ImmutableSet.Builder<Method> setBuilder =
+            new ImmutableSet.Builder<Method>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)